canu/2.1.1/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Use ubuntu as base image
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="Canu"
+LABEL software.version="2.1.1"
+LABEL dockerfile.version="1"
+LABEL description="Long read assembly and polishing tools"
+LABEL website="https://github.com/marbl/canu"
+LABEL license="https://github.com/marbl/canu/blob/master/README.license.GPL"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
```

Comment:
please add your name and email (if you feel comfortable sharing this info). Not required. But nice to have in case someone has a question or issue with this container

```
LABEL maintainer2.name="MillironX"
LABEL maintainer2.email="fake@email.com"
```

Reply:
Fixed by 6dda998b1730c8cf9e9f5772226542b163064291